services/user-management/src/main/resources/application.properties

```diff
@@ -20,4 +21,10 @@ eureka.client.serviceUrl.defaultZone=http://localhost:8761/eureka
 management.tracing.sampling.probability=1.0
 logging.pattern.level=%5p [${spring.application.name},%X{X-B3-TraceId:-},%X{X-B3-SpanId:-},%X{X-Span-Export:-}] %c{1.} : %m%n
 
-spring.config.import=optional:configserver:
+application.security.jwt.secret-key=404E635266556A586E3272357538782F413F4428472B4B6250645367566B5970
```

Comment:
In Prod, you'd want to find a way to secure this value

Reply:
Aligned. There is no way to use a .env file in java or is that possible

Reply:
I'm sure there is a way. I haven't explored it recently, but you can do everything in Java